pkgs/top-level/python-packages.nix

```diff
@@ -2772,6 +2772,8 @@ in {
 
   future = callPackage ../development/python-modules/future { };
 
+  futures = throw "futures is only available for Python 2";
```

Comment:
these throws should be moved to `python2-packages.nix`. You can then write `futures = disabled super.futures;`.

Reply:
Downside is this still evaluates `super.futures`.

Reply:
If I move them to `python2-packages.nix` they won't be available for Python 3 right? The point of these definitions is that they define these attributes for Python 3.

Reply:
You are right, I mixed up 2 and 3.

It's a pity this is needed. The idea is to get rid of all mention of Python 2 from `python-packages.nix`. Contributors should not need to think about Python 2. If we now need to start adding `throw` when a package is not available for Python 3, the whole splitting into `python-packages.nix` and `python2-packages.nix` has been for nothing.

Reply:
We could introduce a mechanism that automatically mirrors all attributes to Python 3 with such a throw